src/array/cuda/spmat_op_impl_csr.cu

```diff
@@ -507,10 +608,16 @@ CSRMatrix CSRSliceMatrix(
   }
 
   // Execute SegmentMaskColKernel
-  int nb = (nnz_csr + nt - 1) / nt;
+  const int64_t num_rows = csr.num_rows;
+  constexpr int WARP_SIZE = 32;
+  constexpr int BLOCK_WARP = 256 / WARP_SIZE;
+  constexpr int TILE_SIZE = 16;
```

Comment:
How did you choose the value of `TILE_SIZE`?

Reply:
When TILE_SIZE = 16, I get the best performance with my device.

Reply:
Add a detailed comment about those magic numbers. If needed put evidence in a google doc, and link the doc to the code.